### PR TITLE
Changed Unknown to Wikidata Description

### LINKED ISSUE #5690: Change `Unknown` on pointer click to ... (Nearby Activity)
Since on the pointer click in the Nearby Activity shows description as Unknown which is pretty confusing, We should probably change it to something else. Suggestions are welcome :-)

WhatsApp Image 2024-04-14 at 11 38 03_c902a97b

### PR DESCRIPTION
**Pull request by kanahia1**
**Description (required)**

Fixes #5690 

What changes did you make and why?

**Tests performed (required)**

Tested prodDebug on Samsung S21 FE with API level 33.

**Screenshots (for UI changes only)**

WhatsApp Image 2024-04-19 at 17 20 36_93e29eef

### CODE REVIEW

File: app/src/main/java/fr/free/nrw/commons/nearby/fragments/NearbyParentFragment.java [hunk 1]
```diff
@@ -1813,7 +1815,11 @@ private void addMarkersToMap(List<BaseMarker> nearbyBaseMarkers) {
             GeoPoint point = new GeoPoint(
                 nearbyBaseMarkers.get(i).getPlace().location.getLatitude(),
                 nearbyBaseMarkers.get(i).getPlace().location.getLongitude());
-            OverlayItem item = new OverlayItem(nearbyBaseMarkers.get(i).getPlace().name, null,
+            OverlayItem item = new OverlayItem(nearbyBaseMarkers.get(i).getPlace().name,
+                containsParentheses(nearbyBaseMarkers.get(i).getPlace().getLongDescription())
+                    ? getTextBetweenParentheses(
```
Comment (nicolas-raoul) [inline app/src/main/java/fr/free/nrw/commons/nearby/fragments/NearbyParentFragment.java:1820]: I believe we get the information without parenthesis from Wikidata, and put the parenthesis ourselves.
I would suggest to modify the place object to retain the information that we can use directly.

Comment (kanahia1) [reply app/src/main/java/fr/free/nrw/commons/nearby/fragments/NearbyParentFragment.java:1820]: Hey tried logging the information/places received through query in `OkHttpJsonApiClient.java` here are the results
`
[Place{name='Indian Institute of Technology Patna', lang='en', label='UNKNOWN', longDescription='Indian Institute of Technology Patna (autonomous institute of education and research in science, engineering and technology located in Patna, Bihar)', location='lat/lng: (25.535675,84.851180555)', category='IIT Patna', distance='null', siteLinks='Sitelinks{wikipediaLink='https://en.wikipedia.org/wiki/IIT_Patna', commonsLink='https://commons.wikimedia.org/wiki/Category:IIT_Patna', wikidataLink='http://www.wikidata.org/entity/Q3440945'}', pic='', exists='true'}, Place{name='Anwesha', lang='en', label='UNKNOWN', longDescription='Anwesha (annual cultural festival at IIT, Patna)', location='lat/lng: (25.53567472,84.85117778)', category='', distance='null', siteLinks='Sitelinks{wikipediaLink='https://en.wikipedia.org/wiki/Anwesha', commonsLink='', wikidataLink='http://www.wikidata.org/entity/Q17049325'}', pic='', exists='true'}, Place{name='State Bank of India, Iit Bihta branch', lang='en', label='UNKNOWN', longDescription='State Bank of India, Iit Bihta branch', location='lat/lng: (25.54079,84.851)', category='', distance='null', siteLinks='Sitelinks{wikipediaLink='', commonsLink='', wikidataLink='http://www.wikidata.org/entity/Q65977647'}', pic='', exists='true'}, Place{name='Mother's Institute of Technology, Bihta', lang='en', label='UNKNOWN', longDescription='Mother's Institute of Technology, Bihta (Engineering college in Bihar)', location='lat/lng: (25.5520451,84.8614291)', category='', distance='null', siteLinks='Sitelinks{wikipediaLink='https://en.wikipedia.org/wiki/Mother%27s_Institute_of_Technology,_Bihta', commonsLink='', wikidataLink='http://www.wikidata.org/entity/Q86753362'}', pic='', exists='true'}, Place{name='SPOT ON 73685 Hotel Sri Hari', lang='en', label='HOTEL', longDescription='SPOT ON 73685 Hotel Sri Hari (hotel in Patna, India)', location='lat/lng: (25.557617199,84.868722762)', category='', distance='null', siteLinks='Sitelinks{wikipediaLink='', commonsLink='', wikidataLink='http://www.wikidata.org/entity/Q111074423'}', pic='', exists='true'}]
`

I guess information is received in the parenthesis, Can you please have a look at it?

Comment (nicolas-raoul) [reply app/src/main/java/fr/free/nrw/commons/nearby/fragments/NearbyParentFragment.java:1820]: Ah I was mistaken then, sorry my bad!
Would you mind checking what comes for Japanese and Arabic locales? The parenthesis might be `（` in Japanese for instance.
Thanks! :-)

Comment (kanahia1) [reply app/src/main/java/fr/free/nrw/commons/nearby/fragments/NearbyParentFragment.java:1820]: Here is the data for ja

`
Place{name='Quad', lang='ja', label='UNKNOWN', longDescription='Quad (Japanese animation studio)', location='lat/lng: (35.691565238,139.719344595)', category='', distance='null', siteLinks='Sitelinks{wikipediaLink='',commonsLink='',wikidataLink='http://www.wikidata.org/entity/Q124512318'}', pic='', exists='true'}, Place{name='株式会社ビープラウド', lang='ja', label='UNKNOWN', longDescription='株式会社ビープラウド (2006年設立の会社)', location='lat/lng: (35.731382366,139.713968947)', category='', distance='null', siteLinks='Sitelinks{wikipediaLink='',commonsLink='',wikidataLink='http://www.wikidata.org/entity/Q125393987'}', pic='http://commons.wikimedia.org/wiki/Special:FilePath/Hhstyle.com.jpg', exists='true'},
, Place{name='東横のれん街', lang='ja', label='UNKNOWN', longDescription='東横のれん街 (東京都渋谷区の商業施設)', location='lat/lng: (35.658944444,139.703416666)', category='', distance='null', siteLinks='Sitelinks{wikipediaLink='https://ja.wikipedia.org/wiki/%E6%9D%B1%E6%A8%AA%E3%81%AE%E3%82%8C%E3%82%93%E8%A1%97', commonsLink='', wikidataLink='http://www.wikidata.org/entity/Q125470413'}', pic='', exists='true'}
`

I think description is received within the parenthesis and name outside the parenthesis

Comment (nicolas-raoul) [reply app/src/main/java/fr/free/nrw/commons/nearby/fragments/NearbyParentFragment.java:1820]: Yes, the same parenthesis, thanks! :-)

**Review by nicolas-raoul — APPROVED** [review]
Working great, thanks! :-)

**Pull request merged**